admin-api-server/src/nft/params.ts

```diff
@@ -0,0 +1,26 @@
+import { IsString, IsInt, IsOptional, IsArray } from 'class-validator';
+import { Type, Transform } from 'class-transformer';
+
+export class NftPaginationParams {
+  @IsInt()
+  @Type(() => Number)
+  @IsOptional()
+  page: number = 1;
+
+  @IsInt()
+  @Type(() => Number)
+  @IsOptional()
+  pageSize: number = 10;
+
+  @IsArray()
+  @Transform(({ value }) => (value ? parseStringArray(value, ',') : undefined))
+  @IsOptional()
+  nftStates?: string[];
+}
```

Comment:
Tiny detail here but is the '?' needed as we are using @IsOptional() and maybe we could use @IsNotEmpty() in combination as well ?

Reply:
the ? is about typescript's typesystem, the @ are about the class-transformer library. maaaybe The class-transformer library is looking at the types of typescript to figure out things like @IsOptional automatically, but I am not sure and the docs aren't that great to figure this out with certainty.

Therefore would rather be verbose in both parts and assume neither looks at the other to deduce things automatically.

`@isnotempty()` seems unrelated? It would actually mean the opposite as far as I understand, it'd require this variable to be set to something other than null. We do want null as a possible value here.